pixie/targets/common.py

```diff
@@ -63,12 +63,16 @@ def parse_target_info_str(data):
 
     all_cpus = parse_target_info_str(target_info_str)
 
-    return CPUEnum('cpus', all_cpus)
+    def fix_dash(x):
+        # Should this really be here?
+        return x.replace('-', '_')
+
+    return CPUEnum('cpus', list(map(fix_dash, all_cpus)))
 
 
 def display_cpu_names():
     triple = llvm.get_process_triple()
-    print(f"Valid CPU names based on the current process LLVM triple: {triple}")
+    print(f"Valid CPU name.replace('_', '-')s based on the current process LLVM triple: {triple}")
```

Comment:
```suggestion
    print(f"Valid CPU names based on the current process LLVM triple: {triple}")
```
find-and-replace mistake?

Reply:
oops